File: 백준/Bronze/4153. 직각삼각형/직각삼각형.js
let input = require("fs").readFileSync("/dev/stdin").toString().split("\n");

for(let i=0; i<input.length-1; i++){
    let [a,b,c] = input[i].split(" ").map((el) => parseInt(el)).sort((a,b) => a-b);
    if(a === 0 && b === 0 && c === 0){
        break;
    }
    else if(a**2 + b**2 === c**2){
        console.log("right");
    }
    else{
        console.log("wrong");
    }
}

File: 백준/Bronze/4153. 직각삼각형/직각삼각형.py
while(True):
    A,B,C = map(int,input().split())
    if A == 0 and B == 0 and C == 0:
        break

    if (A**2 + B**2) == C**2 or (B**2 + C**2) == A**2 or (A**2 + C**2) == B**2:
        print("right")
    else:
        print("wrong")